src/js/notifications.js

```diff
@@ -0,0 +1,67 @@
+var Wand = (function(wand) {
+
+/* Inspiration taken from
+   https://github.com/cugos/dropchop/blob/master/src/js/notifications/Notifications.js
+ */
+
+  'use strict';
+
+  wand = wand || {};
+  wand.notifications = {};
+  var params;
+
+  wand.notifications.init = function() {
```

Comment:
I think as long as we are being consistent, then that's okay. Building the required DOM element in the appropriate file works for me - let's do it like this more often.